работа в pgadmin, перенос в файл idea

Ringonoringo opened the pull request.

Ringonoringo committed:
работа в pgadmin, перенос в файл idea

sproshchaev commented on src/main/java/ru/nabuhiro/java/basic/homeworks/lesson24/questiontest.sql:
```diff
@@ -0,0 +1,45 @@
+    (
+        answer_text text COLLATE pg_catalog."default" NOT NULL,
+        is_true boolean NOT NULL,
+        answer_id integer NOT NULL DEFAULT nextval('answer_answer_id_seq'::regclass),
```
Рекомендуется исправить связь так, чтобы каждый ответ ссылался на соответствующий вопрос через колонку question_id, а не answer_id. Это позволит правильно связывать ответы с вопросами.

Так же можно добавить индекс на поле question_id в таблице answer, так как это улучшит производительность при запросах, связанных с соединением вопросов и ответов.

Ringonoringo committed:
корректировки
добавила в таблицу ответов id вопроса
по другому составила ссылки
убрала лишнее от интерфейса посгресс
